src/shell/command_helper.h

```diff
@@ -709,30 +709,68 @@ inline std::vector<dsn::http_result> get_metrics(const std::vector<node_desc> &n
     return results;
 }
 
+// Adapt the result returned by `get_metrics` into the structure that could be processed by
+// `remote_command`.
+template <typename... Args>
+inline std::pair<bool, std::string> process_get_metrics_result(const dsn::http_result &result,
```

Comment:
It seems the return type can be replaced by `error_s` or `absl::Status`.

Reply:
OK, use `error_s` instead.